docs/source/runningjobs/interactiveandbatch.md

```diff
@@ -3,72 +3,101 @@
 
 In our High-Performance Computing (HPC) environment, users can run jobs in two primary modes: Interactive and Batch. This page provides an in-depth guide to both, assisting users in selecting the appropriate mode for their specific tasks.
 
-## Interactive Mode
+(using-srun)=
+## Interactive Jobs: `srun` Command
+The `srun` command is used to submit an interactive job, which runs in a shell terminal. This method is useful when you want to test a short computation or run an interactive session like a shell, Python, or an R terminal.
 
-Interactive mode allows users to run jobs that need immediate execution and feedback.
+To start an `srun` session, the following syntex is used from the login node:
 
-### Getting Started with Interactive Mode
+:::{code} bash
+srun [options] [command]
+:::
 
-To launch an interactive session, use the following command:
+The following is an example of an `srun` job running on 1 node, with 1 task:
 
 :::{code} bash
-# Request an interactive session
-srun --pty /bin/bash
+srun -N 1 -n 1 -p short --pty bash
 :::
 
-This command allocates resources and gives you a shell prompt on the allocated node.
+- **`n, --ntasks=<number>`**: specify the number of tasks
+- **`N, --nodes=<minnodes[-maxnodes]>`**: specify the number of nodes
+- **`p, --partition=<partition-name>`**: specify a partition for the job to run on
 
-(interactive-mode-use-cases)=
-### Interactive Mode Use Cases
-- **Development and Testing**: Ideal for code development and testing.
-- **Short Tasks**: Best for tasks that require less time and immediate results.
+To see all options for `srun`, please refer to [srun manual] from Schedmd.
 
+### Examples Using `srun`
 
-## Batch Mode
-Batch mode enables users to write scripts that manage job execution, making it suitable for more complex or longer-running jobs.
+You can tailor your request to fit both the needs of the job and the partition limits if you're familiar with the available hardware and partitions on Discovery. 
 
-### Creating Batch Scripts
-A typical batch script includes directives for resource allocation, job names, and commands. Here is an example:
+To request one node and one task for 30 minutes with X11 forwarding on the short partition, type:
```

Comment:
X11 forwarding is a bit tricky, because we assume here that the user connected with `ssh -XY` here. Not sure if we should mention this here, link to "Connecting to the cluster" pages and mention to go through X11 forwarding info therein, or not mention x11 forwarding in this example.

Reply:
Good catch. I added a hyperlink to the x11 for mac (its the same for linux, and automatic for mobaxterm).